podium-gateway/src/main/webapp/app/request/main-detail/detail/request-detail.component.html

```diff
@@ -12,54 +12,54 @@
     <div [ngSwitch]="isInRevision">
         <div *ngSwitchCase="false">
             <dl>
-                <dt><span jhiTranslate="request.detail.title">Title</span></dt>
+                <dt><span [translate]="'request.detail.title'"></span></dt>
                 <dd>
                     <strong><span class="test-title">{{requestDetails.title}}</span></strong>
                 </dd>
-                <dt><span jhiTranslate="request.detail.organisation">Organisation</span></dt>
+                <dt><span [translate]="'request.detail.organisation'"></span></dt>
                 <dd>
                     <span class="test-organisations" *ngFor="let organisation of request.organisations">
-                        {{ organisation.name | capitalize }}
+                        {{ organisation.name | titlecase }}
```

Comment:
Title case is not the same as capitalise. In the case of organisation name, these are not needed anyway.
```suggestion
                        {{ organisation.name }}
```